webnn/idlharness.https.any.js

```diff
@@ -0,0 +1,40 @@
+// META: global=window,worker
+// META: script=/resources/WebIDLParser.js
+// META: script=/resources/idlharness.js
+// META: script=https://webmachinelearning.github.io/webnn-polyfill/dist/webnn-polyfill.js
```

Comment:
Could you add a README.md to explain what is webnn-polyfill and why/how we use it in wpt for now? Thus would help other contributors get started.

Reply:
I think we should land this *without* the polyfill - WPT is meant to test browser implementations, and the polyfill would hide any gap from browser implementations.

(naturally, this means the tests will fail all the browsers at the moment, but that's a proper reflection of the reality of the implementation landscape)

We could also have a separate open PR with just the polyfill added which would allow us to use WPT.fyi to collect results once the polyfill is added.

Reply:
Thanks @dontcallmedom.

> We could also have a separate open PR with just the polyfill added which would allow us to use WPT.fyi to collect results once the polyfill is added.

Do you mean that we need add `webnn-polyfill.js` directly into WPT in the separate PR?  If so, we have to consider maintaining it.

Reply:
what I meant is that we could have a separate PR - not meant to be merged (maybe marked as draft?) which just adds exactly the same line
` // META: script=https://webmachinelearning.github.io/webnn-polyfill/dist/webnn-polyfill.js `

then wpt.fyi would generate results for the said PR which would allow us to get automated results on how browsers behave when the polyfill is added, without polluting the default branch of WPT.